po/he.po

```diff
@@ -2095,7 +2095,7 @@ msgid ""
 "has, though <kbd>--get-price-quotes</kbd> is deprecated in  favor of "
 "<kbd>gnucash-cli --quotes get</kbd>."
 msgstr ""
-"תוכנית GUI, <kbd> gnucash </kbd>, מגיבה עדיין לאפשרויות שהיו זמינות תמיד, אם "
+"תוכנית GUI, <kbd> גנוקאש </kbd>, עדיין מגיבה לאפשרויות שהיו זמינות תמיד, אם "
```

Comment:
@avma 
1. Is there a reason to insert spaces between the surrounding tag pair and its content?
2. Do you enter the command in hebrew letters? If yes, it should be replaced in the other commands, too.